sr_ethercat_hand_config/launch/sr_rosbag_log.launch

```diff
@@ -1,43 +0,0 @@
-<launch>
-  <arg name="split_logs_by_duration" default="True"/>
-  <arg name="log_split_duration" default="10m"/>
-
-  <arg name="split_logs_by_size" default="false"/>
-  <arg name="log_split_size" default="1024"/>
-
-  <arg name="log_max_splits" default="6"/>
-  <arg name="log_directory" default="."/>
-  <arg name="log_bag_prefix" default="sr_log"/>
-
-  <arg name="log_location" value="$(arg log_directory)/$(arg log_bag_prefix)"/>
-
-  <arg name="clear_old_sessions" default="true"/>
-
-  <arg name="log_topics" default='-e ".*(controller|debug_etherCAT_data).*" /joint_states /tf /tf_static /rosout /diagnostics /mechanism_statistics'/>
-
-  <arg name="save_params" default="true"/>
-  <arg name="diff_ws" default="true"/>
-
-  <group unless="$(arg split_logs_by_size)">
-    <node  if="$(arg split_logs_by_duration)" pkg="rosbag" type="record"
-           args='-o $(arg log_location) --lz4 --split --max-splits=$(arg log_max_splits) --duration $(arg log_split_duration) $(arg log_topics)' name="record"/>
-  </group>
-
-  <group unless="$(arg split_logs_by_duration)">
-    <node if="$(arg split_logs_by_size)" pkg="rosbag" type="record"
-          args='-o $(arg log_location) --lz4 --split --max-splits=$(arg log_max_splits) --size $(arg log_split_size) $(arg log_topics) ' name="record"/>
-  </group>
-
-  <node if="$(arg clear_old_sessions)" pkg="sr_ethercat_hand_config" type="bag_rotate.py" name="bag_rotate">
-    <param name="bag_files_num" value="$(arg log_max_splits)"/>
-    <param name="bag_files_path" value="$(arg log_directory)"/>
-  </node>
-
-  <node if="$(arg save_params)" pkg="sr_ethercat_hand_config" type="record_params.py" name="param_dump">
-    <param name="log_directory" value="$(arg log_directory)" />
-  </node>
-
-  <node if="$(arg diff_ws)" pkg="sr_ethercat_hand_config" type="ws_diff.py" name="ws_diff">
-    <param name="log_directory" value="$(arg log_directory)"/>
-  </node>  
-</launch>
```

Comment:
You also need to modify all the launchfiles in https://github.com/shadow-robot/sr-config/tree/kinetic-devel/sr_ethercat_hand_config/launch
that use the sr_rosbag_log.launch, so that they use the copy in sr_logging_common

And you also need to add a run dependency on sr_logging_common to package.xml

Reply:
Not sure I have add the run dependency correctly so please make sure you check that.